src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -180,13 +229,13 @@ class ActionsPanel extends React.PureComponent<Props> {
             })}
           <PanelItem>
             <StyledSelectControl
-              deprecatedSelectControl
               name="add-action"
               aria-label={t('Add an Action')}
-              disabled={disabled || loading}
+              isDisabled={disabled || loading}
               placeholder={t('Add an Action')}
               onChange={this.handleAddAction}
-              options={items}
+              value={null}
```

Comment:
We always want this to be null so that placeholder always shows, it's not really a Select component but more of a dropdown.

Reply:
Well, we want the placeholder to tell the user what to do, and it's intentional that we never have a value selected because we use the on change event to add a new action.

Reply:
Can we make it null by default so we don't need this prop?

Reply:
Without a `value` property, the Select component keep whatever action you try to add as the "selected" value. E.g. you add a slack action, it would keep that selected. The desired behavior is to always have nothing selected.